test/dynamo/test_export.py

```diff
@@ -1426,11 +1426,13 @@ def __init__(self):
                 self.linear = torch.nn.Linear(3, 3)
 
             def forward(self, pred, x):
+                y = x + x
+
                 def true_fn(val):
-                    return self.linear(val) * torch.tensor(2)
+                    return self.linear(val) * (x + y)
 
                 def false_fn(val):
-                    return self.linear(val) * torch.tensor(-1)
+                    return val * (y - x)
```

Comment:
It seems better to make this a different test rather than cannibalize the old one